### PR TITLE
lucene 8.5

### PR DESCRIPTION
**Pull request by tarzanek**

### CODE REVIEW

**Comment by tarzanek** [comment]
it seems windows build fails in https://travis-ci.org/github/tarzanek/OpenGrok/jobs/668662020
@vladak are windows builds working OK?

File: opengrok-web/src/main/java/org/opengrok/web/api/v1/suggester/query/SuggesterQueryParser.java [hunk 1]
```diff
@@ -93,14 +93,14 @@ public String getQueryTextWithPlaceholder() {
     }
 
     @Override
-    protected Query newTermQuery(final Term term) {
+    protected Query newTermQuery(final Term term, float boost) {
         if (term.text().contains(identifier)) {
             SuggesterPrefixQuery q = new SuggesterPrefixQuery(replaceIdentifier(term, identifier));
             this.suggesterQuery = q;
```
Comment (idodeclare) [inline opengrok-web/src/main/java/org/opengrok/web/api/v1/suggester/query/SuggesterQueryParser.java]: Probably `boost` should be checked to see if it's applicable here:

```
    if (boost != DEFAULT_BOOST) {
        q = new BoostQuery(q, boost);
    }
```

**Comment by vladak** [comment]
> it seems windows build fails in https://travis-ci.org/github/tarzanek/OpenGrok/jobs/668662020
> @vladak are windows builds working OK?

Hah. Windows builds have been experiencing intermittent failures for quite some time :-) PR #3086 should fix this - you are welcome to review.

**Comment by vladak** [comment]
Just to confirm - no reindex needed ?

**Comment by tarzanek** [comment]
nope, no reindex, let me fix what Chris suggested

**Comment by tarzanek** [comment]
hmm, checkstyle :-D OMG

**Comment by idodeclare** [comment]
😲 🤣

**Comment by tarzanek** [comment]
I guess 8.5.1 will be out soon, so let's wait with merging of this

**Comment by tarzanek** [comment]
8.5.1 is now pushed here

**Pull request merged**